CCDB/include/CCDB/Condition.h

```diff
@@ -109,6 +110,19 @@ class Condition : public TObject
       return mObject;
     };
 
+    /// retrieve object as specified type
+    /// return false if failed/true otherwise
+    template<typename T>
+    bool getObjectAs(T *& obj) {
+      obj = nullptr;
+      // test if saved object was a wrapped object
+      if(auto wrapped = dynamic_cast<TObjectWrapper<T>*>(mObject)) {
```

Comment:
Do we need to preserve any backward compatibility? By doing this change everything is expected to use `TObjWrapper`, right?

Reply:
This is a new API next to the existing one so backward compatibility is preserved.